packages/common/src/types/command/PartialTargetDescriptor.types.ts

```diff
@@ -43,14 +43,31 @@ export interface RangeMark {
   excludeActive?: boolean;
 }
 
+interface SimplePosition {
+  readonly line: number;
+  readonly character: number;
+}
+
+interface SimpleRange {
+  readonly start: SimplePosition;
+  readonly end: SimplePosition;
+}
+
+export interface ExplicitRangeMark {
+  type: "explicitRange";
+  editorId: string;
+  range: SimpleRange;
+}
+
 export type PartialMark =
   | CursorMark
   | ThatMark
   | SourceMark
   | DecoratedSymbolMark
   | NothingMark
   | LineNumberMark
-  | RangeMark;
+  | RangeMark
+  | ExplicitRangeMark;
```

Comment:
oh sorry i forgot we have a `RangeMark` already.  A bit confusing to have both, esp since we have something called `hasExplicitRange`, and this name has nothing to do with that 😅.  Maybe let's discuss.  Sorry for the needless work

Reply:
Good catch. We can have a talk about it.

Reply:
update: go back to `ExplicitMark` and `"explicit"`, but add jsdoc